### PR TITLE
Warning fix

### PR DESCRIPTION
**Pull request by ChunMinChang**
Fix warning on Travis CI

### CODE REVIEW

File: src/cubeb_wasapi.cpp [hunk 1]
```diff
@@ -831,15 +831,15 @@ refill_callback_output(cubeb_stream * stm)
        output_frames, got);
 
   XASSERT(got >= 0);
-  XASSERT(got == output_frames || stm->draining);
+  XASSERT((unsigned long)got == output_frames || stm->draining);
```
Comment (kinetiknz) [inline src/cubeb_wasapi.cpp]: Space between ) and got here and everywhere else there's a cast.

File: src/cubeb_wasapi.cpp [hunk 2]
```diff
@@ -795,7 +795,7 @@ refill_callback_input(cubeb_stream * stm)
                      nullptr,
                      0);
 
-  consumed_all_buffer = read == stm->linear_input_buffer.length();
+  consumed_all_buffer = (unsigned long)read == stm->linear_input_buffer.length();
```
Comment (kinetiknz) [inline src/cubeb_wasapi.cpp]: I think we need an `XASSERT(read >= 0)` before this to ensure we're not casting negative values into an unsigned type.

Comment (ChunMinChang) [inline src/cubeb_wasapi.cpp]: ok.

File: src/cubeb_wasapi.cpp (hunk 1 above)
Comment (ChunMinChang) [inline src/cubeb_wasapi.cpp]: ok.

**Comment by kinetiknz** [comment]
Thanks! I merged this via a squash in https://github.com/kinetiknz/cubeb/commit/b0f1a6331f327b4fc127c8d206bca944f1141b80.

**Comment by ChunMinChang** [comment]
Thanks for the help and correction about my wrong fix for resampler.c. You save the code! 
I'll try upstreaming the fixes on its own repo!

**Comment by kinetiknz** [comment]
No problem, thanks for the patches! Let me know if I can help with the speexdsp upstreaming at all.

**Pull request closed without merging**